tests/regression/test/HalmosCheatCode.t.sol

```diff
@@ -138,6 +138,148 @@ contract HalmosCheatCodeTest is SymTest, Test {
         // symbolic storage is not allowed for a nonexistent account
         svm.enableSymbolicStorage(address(0xdeadbeef)); // HalmosException
     }
+
+    function check_createCalldata_Beep_1_excluding_pure() public {
+        bytes memory data = svm.createCalldata("HalmosCheatCode.t.sol", "Beep");
+        _check_createCalldata_Beep(data); // fail // not reachable, no calldata generated above
+    }
```

Comment:
it is expected to fail because the only function is pure and it is excluded in the calldata generation.

agree that the error message needs to be improved. it should say something like "no success (or normally terminated) path exists."